tests/goerli/chronos/traveler.handler.test.ts

```diff
@@ -0,0 +1,33 @@
+import { assert, describe, test, newMockEvent, beforeEach, afterEach, clearStore } from 'matchstick-as'
+import { handleTravelerTransfer } from '../../../src/chronos/traveler/handler'
+import { generateId } from '../../../src/utils/helper'
+import { createNewTransferEvent } from '../../mocks/events'
+import { createMockNft } from '../../mocks/entities'
+import { ZERO_ADDRESS } from '../../../src/utils/constants'
+
+const tokenId = '123'
+const address1 = '0x1111111111111111111111111111111111111111'
+const address2 = '0x2222222222222222222222222222222222222222'
+
+describe('When entities no exists', () => {
+  beforeEach(() => {
+    createMockNft(tokenId)
+  })
```

Comment:
Since tranfer handler also is responsible to create the `Nft` entity. Maybe would be nice to not create a mock here, and use `assert.entityCount` before calling `handleTravelerTransfer` to make sure that the handler is also taking care of creating the NFT